src/calc-methods/index.ts

```diff
@@ -38,6 +44,22 @@ export const getCalcMethodsByCountry = (
   };
 };
 
+export const getCalculationMethodByCountry = (
```

Comment:
don't we have same method above? 
https://github.com/prayersconnect/configs/pull/50/files#diff-bd804961d106f9828ce84d7648ccef64bf9453731e7d3366b46b2c6fd93c3a7aR33

Reply:
this concerns me as it will cause confusion of a module having two different methods of seemingly same purpose. 

unless you can think of a cleaner approach, consider overloading

https://learntypescript.dev/05/l3-overloading

Reply:
I do agree method overloading with be better, but since the method would be deprecated due to the calculation method names will no longer work in the future, I can add a @deprecated message 
```
/**
 * @deprecated The method should not be used
 */
 ```